.coveragerc

```diff
@@ -0,0 +1,47 @@
+# .coveragerc for edx-platform
+[run]
+data_file = reports/.coverage
+source =
+    cms
+    common/djangoapps
+    common/lib/calc
+    common/lib/capa
+    common/lib/xmodule
+    lms
+    openedx/core/djangoapps
+
+omit =
+    cms/envs/*
+    cms/manage.py
+    cms/djangoapps/*/migrations/*
+    cms/djangoapps/*/features/*
+    lms/envs/*
+    lms/djangoapps/*/migrations/*
+    lms/djangoapps/*/features/*
+    common/djangoapps/terrain/*
+    common/djangoapps/*/migrations/*
+    openedx/core/djangoapps/*/migrations/*
+
+branch = true
+
+[report]
+ignore_errors = True
+
+exclude_lines =
```

Comment:
Note:  These `exclude_lines` were originally included in `common/lib/xmodule/.coveragerc` only.  Up for discussion about whether or not they should be included.